src/main/java/net/mcreator/gradle/GradleUtils.java

```diff
@@ -118,6 +121,17 @@ public static void updateMCreatorBuildFile(Workspace workspace) {
 		}
 	}
 
+	public static void removeStoreFiles(File folder) {

```

Comment:
Rename to removeDSStoreFiles.

Also do OS check so other operating systems don't have to unnecessarily walk through all the files.